config/SOUE01/rels/d_t_gate_to_groundNP/symbols.txt

```diff
@@ -2,15 +2,15 @@ _prolog = .text:0x00000000; // type:function size:0x2C scope:global
 _epilog = .text:0x00000030; // type:function size:0x2C scope:global
 _unresolved = .text:0x00000060; // type:function size:0x4 scope:global
 dTgGateToGround_c_classInit__Fv = .text:0x00000070; // type:function size:0x5C
-TgGateToGround__init = .text:0x000000D0; // type:function size:0x7C
-fn_205_150 = .text:0x00000150; // type:function size:0x8
-TgGateToGround__update = .text:0x00000160; // type:function size:0x138
-TgGateToGround__updateInEvent = .text:0x000002A0; // type:function size:0x44
-fn_205_2F0 = .text:0x000002F0; // type:function size:0x8
+create__17dTgGateToGround_cFv = .text:0x000000D0; // type:function size:0x7C
+doDelete__17dTgGateToGround_cFv = .text:0x00000150; // type:function size:0x8
+actorExecute__17dTgGateToGround_cFv = .text:0x00000160; // type:function size:0x138
+actorExecuteInEvent__17dTgGateToGround_cFv = .text:0x000002A0; // type:function size:0x44
+draw__17dTgGateToGround_cFv = .text:0x000002F0; // type:function size:0x8
 __dt__17dTgGateToGround_cFv = .text:0x00000300; // type:function size:0x70
 _ctors = .ctors:0x00000000; // type:label scope:global
 _dtors = .dtors:0x00000000; // type:label scope:global
-lbl_205_rodata_0 = .rodata:0x00000000; // type:object size:0x4 data:float
+@20562 = .rodata:0x00000000; // type:object size:0x4 data:float
```

Comment:
Nothing wrong with renaming these `@` symbols too, but the numbers will change in the future with unrelated changes in headers anyway. The best thing to do here is to add a `scope:local` at the end (no renaming necessary) so that the linker will stop complaining (I also wrote this down at https://wiki.decomp.dev/en/projects/gamecube-wii/skyward-sword#how-to-deal-with-local-anonymous-symbols for the future).

The vtable (`__vt__`) rename is good though.